app/src/main/res/xml/status_widget_info.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:description="@string/status_widget_description"
+    android:initialKeyguardLayout="@layout/status_widget"
+    android:initialLayout="@layout/status_widget"
+    android:minWidth="40dp"
+    android:minHeight="40dp"
+    android:previewImage="@drawable/status_widget_preview"
```

Comment:
Can we use any of the existing images? If not, can we use vector?

Reply:
I thought, the images would be later customized, and cloned existing ones:
![ic_start](https://user-images.githubusercontent.com/20116984/150692602-4e8865f7-2bcf-46ae-91f8-cb2bb06c3582.png) indicates the running (enabled) state
![ic_stop](https://user-images.githubusercontent.com/20116984/150692612-3f9fede1-124f-4ea0-8705-c6380ce85c9e.png) indicates the stopped (disabled) state
If these images are OK, I can drop the clones.

Reply:
Wait. Can we just show a switch or something instead of these images?

Reply:
Replacing `ImageView` with `Switch` breaks the widget - it fails to render itself in Android 7 (AVD).
```
    <Switch
        android:id="@+id/swEnabled"
        android:layout_width="match_parent"
        android:layout_height="wrap_content" />
```

Reply:
That's why I said "or something". I meant that it makes more sense to render proper views instead of images.

Reply:
Widgets rendering seems to be a bit limited in capabilities: though you can place `Button` there, `ToggleButton` isn't supported (or I'm overlooking something).
I'm not sure which view you want to place there. Can you elaborate on this?
I also tried to use vector drawing (from `ic_stat_notify.xml`), and it seems to work fine, though you have to use `android:tint` to change its color while `app:tint` (as suggested by Android Studio) has no effect. Anyway such one-colored image doesn't look nice :)

Reply:
Anything that looks good. It can be a text field with a button. A checkbox or anything else.

Reply:
Here is how all icons look on Android 9 (HDPI, Amazon Fire HD 10, 1920x1200):
![2022 01 24-Android_9-HDPI-Amazon_Fire_HD_10-1920x1200](https://user-images.githubusercontent.com/20116984/150702409-1874df99-6651-4198-84fb-fb9a25828f74.png)
Another look on Android 6 (XHDPI, Samsung Galaxy S6, 768x1280):
![2022 01 24-Android_6-XHDPI-Samsung_Galaxy_S6-768x1280](https://user-images.githubusercontent.com/20116984/150702608-071b4a69-15e7-4c0e-a104-67b4743daacf.png)
As for my taste they are all of same style :)

Reply:
Looks ok. The only thing I don't like is icon duplication. If we can reuse existing icons, I'm good with this.